health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/CampaignDetails.js

```
@@ -57,6 +57,18 @@ const CampaignDetails = ({ onSelect, props: customProps, ...props }) => {
     { schemaCode: "ProjectType" }
   );
 
+  const { isMicroplanCampignTypesLoading, data : microplanCampignTypes } = Digit.Hooks.useCustomMDMS(
+    tenantId,
+    "HCM-MICROPLANNING-CAMPAIGN-TYPES",
+    [{ name: "MicroplanCamplignTypes" }],
+    {
+      select: (microplanCampignTypes) => {
+        let campaignTypeOptions = microplanCampignTypes?.["HCM-MICROPLANNING-CAMPAIGN-TYPES"]?.MicroplanCamplignTypes;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'campaignTypeOptions' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)